shared/patches/recyclerlistview+4.2.0.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/node_modules/recyclerlistview/dist/reactnative/core/layoutmanager/LayoutManager.js b/node_modules/recyclerlistview/dist/reactnative/core/layoutmanager/LayoutManager.js
+index 9cd3c57..f65012e 100644
+--- a/node_modules/recyclerlistview/dist/reactnative/core/layoutmanager/LayoutManager.js
++++ b/node_modules/recyclerlistview/dist/reactnative/core/layoutmanager/LayoutManager.js
+@@ -114,7 +114,7 @@ var WrapGridLayoutManager = /** @class */ (function (_super) {
+         for (var i = startIndex; i < itemCount; i++) {
+             oldLayout = this._layouts[i];
+             var layoutType = this._layoutProvider.getLayoutTypeForIndex(i);
+-            if (oldLayout && oldLayout.isOverridden && oldLayout.type === layoutType) {
++            if (oldLayout && oldLayout.isOverridden /*&& oldLayout.type === layoutType*/) {
```

Comment:
the relationship between the types has no greater significance here. If its overridden keep the size or it can get lost as its possible it never gets re-added and then fires an `onLayout` losing the size forever